LoopKitUI/Views/DismissibleKeyboardTextField.swift

```diff
@@ -16,21 +16,27 @@ public struct DismissibleKeyboardTextField: UIViewRepresentable {
     var textColor: UIColor
     var textAlignment: NSTextAlignment
     var keyboardType: UIKeyboardType
+    var autocapitalizationType: UITextAutocapitalizationType
+    var autocorrectionType: UITextAutocorrectionType
 
     public init(
         text: Binding<String>,
         placeholder: String,
         font: UIFont = .preferredFont(forTextStyle: .body),
         textColor: UIColor = .label,
         textAlignment: NSTextAlignment = .natural,
-        keyboardType: UIKeyboardType = .default
+        keyboardType: UIKeyboardType = .default,
+        autocapitalizationType: UITextAutocapitalizationType = .allCharacters,
+        autocorrectionType: UITextAutocorrectionType = .no
```

Comment:
Oh, sorry, I missed these. These arguments should be match defaults of UITextField.